labgrid/driver/smallubootdriver.py

```diff
@@ -96,7 +96,10 @@ def _run(self, cmd: str, *, timeout: int = 30, codec: str = "utf-8", decodeerror
         cmp_command = f"echo{marker}; {cmd}; echo{marker}"
 
         self.console.sendline(cmp_command)
```

Comment:
I wonder when the extra characters appear.?Is it before the echo is sent, or after?